src/core_plugins/kibana/public/visualize/listing/visualize_listing.js

```diff
@@ -0,0 +1,167 @@
+import SavedObjectRegistryProvider from 'ui/saved_objects/saved_object_registry';
+import 'ui/pager_control';
+import 'ui/pager';
+import _ from 'lodash';
+
+export function VisualizeListingController(
+  $filter,
```

Comment:
Would you mind trying `const $filter = $injector.get('$filter')` here and below? It's a part of https://github.com/elastic/kibana/pull/9495 that, among other things, prevents argument lists like this